### ISSUE TITLE
terms misusing 'results in specification of'

### ISSUE THREAD

**Issue by balhoff** [geneontology/go-ontology#28016]
We have a couple of terms using [results in specification of](http://purl.obolibrary.org/obo/RO_0002356) in their logical definitions. The range of this property is [cell](http://purl.obolibrary.org/obo/CL_0000000), but we are using it for plant organ specification. The new ELK reasoner catches this problem in the go-lego build, where we have a bridge from plant cell to CL cell. The terms are:

- [specification of floral organ identity](http://purl.obolibrary.org/obo/GO_0010093)
- [specification of plant organ identity](http://purl.obolibrary.org/obo/GO_0090701)

Could we change to [results in structural organization of](http://purl.obolibrary.org/obo/RO_0002355)?

We will need to fix this in order to move to the new Protege and ODK.

**Comment by pgaudet** [comment]
@tberardini Does that work ?

**Comment by tberardini** [comment]
@pgaudet What would the new logical definition look like?

'results in structural organization of' 'floral organ identity' seems odd since floral organ identity is not an entity that is organized

'results in structural organization of' 'floral organ' doesn't quite capture the identity part

Maybe we need to just drop the logical def for these terms. I'm not familiar enough with RO to pick another relation.

**Comment by ukemi** [comment]
Is this a type of regionalization, defining the pattern where the floral organ cells will differentiate. Isn't this specified by homeotic genes? @tberardini, I think we looked at this a looooong time ago when comparing plants and animals?

**Comment by tberardini** [comment]
Yes, floral homeotic genes take care of the specification of identify (petal vs. sepal vs. carpel vs. stamen) see 2014 paper (figure 1): https://academic.oup.com/jxb/article/65/1/1/431211

Depending on which combination of transcription factor complexes are expressed, you get a different floral organ.

**Comment by balhoff** [comment]
I think dropping the logical definitions might be a solution. We could add a weaker relation to the GO terms using the parent property [results in developmental progression of](http://purl.obolibrary.org/obo/RO_0002295) in order to keep the link to the PO terms.

**Comment by tberardini** [comment]
sounds ok to me
